### DISCUSSION TITLE
Not seeing my repository in Featured work

### DISCUSSION THREAD

**Discussion by waterplea** [community/community#46586] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm creating my GitHub sponsor profile and one of my repositories I wanted to showcase is not showing in the list to pick for Featured work:
https://github.com/AngularWave/rxjs-challenge
Is there something I need to enable to be able to showcase it?

**Comment by kaibloecker** [comment]
The repo you link to is not in your user's namespace, and therefore does not show up when selecting one of your repos to showcase.

An alternative could be to create a .github/FUNDING.yml file in the rxjs-challenge repo with the content `github: waterplea`. That way your user's sponsor link will show in the AngularWave/rxjs-challenge.

**Reply by waterplea** [reply]
But other repositories I contribute to that are not in my namespace show up, namely:
https://github.com/taiga-family/taiga-ui
https://github.com/taiga-family/ng-web-apis

**Reply by waterplea** [reply]
Besides [AngularWave](https://github.com/AngularWave) is an Organization I created and manage.

**Comment by lroal** [comment]
I have the same problem. One of the organization repos shows, but not the other twos.

These two don't appear in the drop down:
- https://github.com/alfateam/node-cls
- https://github.com/alfateam/a

But this one does:
- https://github.com/alfateam/rdb
